### PR TITLE
Modified previous summaries, add the new one, the excel file 1st book and the conclusion

### PR DESCRIPTION
**Pull request by Nebhey**
It seems that the last commit have be reset. I do not know why. I may have still trouble to use GitHub and I am sorry for the trouble I made. The modification on the summaries are purely formal so no big deal (and it works apparently)

### CODE REVIEW

**Comment by Nebhey** [comment]
I know now that I have to update the fork before pushing it. I will try to resolve that problem.

**Comment by kbuzard** [comment]
I'm working my way through the documents now. One question and one
clarification:

Clarification: I'll be going to the "logrolling" talk at noon today, but
you don't need to be there unless you are interested. It's more directly
relevant to some of my other work. I'm sure it would give you deeper
insights into the idea of logrolling, but it may not be an efficient way to
get what you need for this project, so I don't want to ask you go to unless
you're independently interested.

Question: in your documents, as long as we can both understand what is
intended, I don't care whether there are typos or turns of phrase that are
not natural English. But let me know if you would prefer that I point these
out to you for your own edification.

--
Kristy Buzard
Assistant Professor of Economics
Syracuse University
110 Eggers Hall, Syracuse, NY 13244-1020
(315) 443-4079
kbuzard@syr.edu
faculty.maxwell.syr.edu/kbuzard

File: Book conclusion, policy and trade politics.Rmd [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+##TAA frictions: 1934-1958 through the trade lens
+
+* The domestic issue: it is historically the first that appears, as the bill is presented by the Roosevelt organization in order help recovery from the Great Depression. In opposition to this position, the protectionist movement insists in the danger that represent the TAA for local producer. This is where local private interest are defended: agriculture (multiple amendments), fur, oil (several times), watch, mining, fishing, wool and textile. The private interest amendment are usually provided by Representatives and Senators who are in direct contact with the concerned parties (South, especially for Democrats).
```
Comment (kbuzard) [inline Book conclusion, policy and trade politics.Rmd]: just before "mining," what does "watch" refer to? The devices we wear on our wrists to tell time?

File: Book conclusion, policy and trade politics.Rmd [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* The trade issue: the bill faces a specific opposition regarding the changes in the trade organization during the period. First, the TAA suffers from the expression of residual protectionist policies. The Smoot-Hawley agreement haunts the Congress until the late 1940s and the opposition to the status of most favored nation find a new application with the beginning of Cold War. Moreover, the birth of the GATT and the project of global organization challenges the content of the TAA. At multiple time, the TAA is linked to the GATT and the adoption a global organization charter (the ITO or the OTC). 
+
```
Comment (kbuzard) [inline Book conclusion, policy and trade politics.Rmd]: Can you elaborate a bit more on this last sentence? How does the GATT come into conflict with the TAA?

File: Book conclusion, policy and trade politics.Rmd [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+At the end of the period, it seems like the forces at stake just balanced. Neither the free-traders nor the protectionist had a serious advantage and the disappearance of the security question as in support for free trade does not enable furthermore liberalization. The question is how the change in the question of security, as Cold War changes from the harsh period to the Detente, will change the tariff distribution.
```
Comment (kbuzard) [inline Book conclusion, policy and trade politics.Rmd]: I think the work might flow better if pull requests were broken up into smaller chunks. This Q&A suggests different branches for (in our case) different documents or types of documents. http://www.quora.com/Is-there-a-good-system-for-adding-multiple-pull-requests-to-GitHub.

**Comment by kbuzard** [comment]
I've finished reviewing the five documents, and just had a few requests for additional detail in the summary of Policy and Trade Politics.

File: Book conclusion, policy and trade politics.Rmd (hunk 1 above)
Comment (Nebhey) [inline Book conclusion, policy and trade politics.Rmd]: yes, Swiss watches to be exact

File: Book conclusion, policy and trade politics.Rmd (hunk 2 above)
Comment (Nebhey) [inline Book conclusion, policy and trade politics.Rmd]: The GATT is seen by the Republican as tightly linked to the TAA and the ITO charter. In 1949, the senator Miliken (major actor in the Republican trade policy) makes a speech where he tries to demonstrate the link between the two. Therefore, the Adminsitration has to assure Congress that the GATT, the ITO charter and the TAA are three completely different stakes.

**Comment by Nebhey** [comment]
I am a student that need to learn and I am happy to do so. So if you do not mind, I will be glad to receive some advice on non-English grammar. I will also try to minimize these problems by carefully reviewing the documents.

**Pull request merged**